src/clock.ml

```diff
@@ -35,6 +35,22 @@ let log = Log.make ["clock"]
 let conf_clock =
   Dtools.Conf.void ~p:(Configure.conf#plug "clock") "Clock settings"
 
+module Time : Liq_time.T = (val !Liq_time.implementation)
```

Comment:
Can't we simply use the configure / Makefile in order to choose the implementation of the module at compilation time, instead of having a this reference?

Reply:
I can see it either way. Let me think about it for a sec.. 🙂

Reply:
I think that this is fine this way. It's more explicit but also it should make it possible to later convert it to a settings if we feel like it.